tests/transition.md

```diff
@@ -277,3 +281,381 @@ All test cases described in this document are beginning in a pre-Merge world and
     * EL eventually syncs `B1 <- ... <- B1024`, and responds `{status: VALID, latestValidHash: Pn, validationError: null}` to `forkchoiceUpdated(head: Pn)` where `B1 <- ... <- B1024 <- P1 <- ... <- Pn`
   
   </details>
+
+* [x] [[Hive](https://github.com/ethereum/hive/blob/32ff59c89c879972f00abe62137a566def81a080/simulators/ethereum/engine/clmock.go#L346)] Propose valid transition block
+  <details>
+  <summary>Click for details &#9662;</summary>
+
+  * `Genesis <- ... <- TB`, `TB` is a valid terminal block
+  * EL starts with `TB` as the head
+  * `forkchoiceUpdated(headBlockHash: TB.blockHash, payloadAttributes: mergeTransitionBlockAttributes)`
+    * EL's head is set to `headBlockHash`
+    * EL returns `{status: VALID, payloadId: mergeTransitionPayloadId}`
+  * `getPayload(mergeTransitionPayloadId)`
+    * EL returns merge transition payload
+      * `ommersHash == 0x1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347`
+      * `difficulty == 0`
+      * `mixHash == mergeTransitionBlockAttributes.prevRandao`
+      * `nonce == 0x0000000000000000`
+  * `newPayload(mergeTransitionPayload)`
+    * EL returns `VALID`
+  
+  </details>
+
+* [x] [[Hive](https://github.com/ethereum/hive/pull/553)] Transition on a valid chain with missed `fcU`
+  <details>
+  <summary>Click for details &#9662;</summary>
+
+  * `Genesis <- ... <- TB <- P1 <- P2 <- P3`, `TB` is a valid terminal block
+  * EL starts with `TB` as the head
+  * `newPayload(P1)`
+    * EL returns `{status: VALID, latestValidHash: payload.blockHash}`
+    * EL's head points to `TB`
+  * `newPayload(P2)`
+    * EL returns `{status: VALID, latestValidHash: payload.blockHash}`
+    * EL's head points to `TB`
+  * `newPayload(P3)`
+    * EL returns `{status: VALID, latestValidHash: payload.blockHash}`
+    * EL's head points to `TB`
+  * `forkchoiceUpdated(head: P3, safe: P2, finalized: P1)`
+    * EL's head updated to `P3`
+    * `eth_getBlockByNumber(safe)` returns `P2`
+    * `eth_getBlockByNumber(finalized)` returns `P1`
+  
+  </details>
+
+* [ ] Build atop of invalid terminal block
```

Comment:
Should be covered by https://github.com/ethereum/hive/blob/f2838a629dcd1a027a260cd10fc84899f89e3ba6/simulators/ethereum/engine/enginetests.go#L410